Add doc files for syncable exports (#1737)
theforeman/foreman-documentation: guides/common/modules/snip_generating-content.adoc

ifdef::satellite[]
You can then serve the generated content using a local web server on the importing {ProjectServer} or in another {ProjectServer} organization.

You cannot directly import Syncable Format exports.
Instead, on the importing {ProjectServer} you must:

* Copy the generated content to an HTTP/HTTPS web server that is accessible to importing {ProjectServer}.
* Update your CDN configuration to *Custom CDN*.
* Set the CDN URL to point to the web server.
* Optional: Set an SSL/TLS CA Credential if the web server requires it.
* Enable the repository.
* Synchronize the repository.
endif::[]

ifndef::satellite[]
You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
For more information, see xref:Importing_a_Content_View_Version_{context}[].
endif::[]

You can export the following content in a syncable format from {ProjectServer}:

* Yum repositories
* Kickstart repositories
* File repositories

> You cannot export Ansible and Docker content.
> 
> You cannot export DEB repositories from {ProjectServer}.

Reviewer: ```suggestion
You cannot export Ansible, DEB, and Docker content.
```

ifdef::satellite[]
You can then serve the generated content using a local web server on the importing {ProjectServer} or in another {ProjectServer} organization.

You cannot directly import Syncable Format exports.
Instead, on the importing {ProjectServer} you must:

* Copy the generated content to an HTTP/HTTPS web server that is accessible to importing {ProjectServer}.
* Update your CDN configuration to *Custom CDN*.
* Set the CDN URL to point to the web server.
* Optional: Set an SSL/TLS CA Credential if the web server requires it.
* Enable the repository.
* Synchronize the repository.
endif::[]

ifndef::satellite[]
You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
For more information, see xref:Importing_a_Content_View_Version_{context}[].
endif::[]

You can export the following content in a syncable format from {ProjectServer}:

> * RPM repositories

Reviewer: ```suggestion
* Yum repositories
```

* Kickstart repositories
* File repositories

You cannot export Ansible and Docker content.

> ifndef::satellite[]
> You cannot export DEB repositories from {ProjectServer}.
> endif::[]

Reviewer: ```suggestion
You cannot export DEB repositories from {ProjectServer}.
```

> The export contains the following data:
> 
> * Directories containing the packages, *listing* files and metadata of the repository in yum format that can be used to synchronize in the importing {ProjectServer}.

Reviewer: ```suggestion
The export contains directories with the packages, *listing* files, and metadata of the repository in Yum format that can be used to synchronize in the importing {ProjectServer}.
```
